Box.V2/Models/Request/BoxSharedLinkRequest.cs

```diff
@@ -33,6 +33,12 @@ public class BoxSharedLinkRequest
         /// </summary>
         [JsonProperty(PropertyName = "password")]
         public string Password { get; set; }
```

Comment:
@coolboy  When I look to documentation, fields Password, Permissions are not required
https://docs.box.com/reference#create-a-shared-link-for-a-folder
Do you think that  I should check that fields are mandatory